### DISCUSSION TITLE
Shortcut for switching difficulties in editor

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#31202] [Ideas]
In stable it's possible to switch to another difficulty in the editor by Ctrl+O. In lazer this feature can only be accessed from the menu (or I did not find the shortcut), which is a bit inconvenient imo. Personally I find it useful when I want to ensure the new difficulty forms a good spread with existing difficulties (e.g. avoid inserting more rhythm in a Hard than an Insane)

**Comment by peppy** [comment]
This will conflict with global settings. In discussions opinions were split over whether it would be okay for this to locally override in the editor. Open to giving it a try though.

**Reply by cihe13375** [reply]
I'm personally leaning towards giving a new shortcut for switching difficulties (if we are not exhausting key combinations...). It's nice to be able to access the settings menu in the editor, and new mappers starting in lazer may get confused if Ctrl+O has a special meaning in the editor. Again that's only my own opinion.

**Comment by Judge1st** [comment]
I wonder if lazer could have a new way to switch between difficulties like "switch to next diff" and "switch to previous diff" hotkeys (based on star rating). While that creates even more hotkeys if there's still some place for those would be interesting to have it (maybe they could be unassigned by default).

I think such hotkeys would be quite useful as usually switching happens between neighboring difficulties (to reference intensity level while creating lower difficulty for example) or switching happens in established order to resolve mods, so host would go through every difficulty anyway. Feels like good improvement to avoid extra clicks and pop out menu.
